frontend/src/composables/credential/useSecretContext.js

```diff
@@ -41,7 +41,7 @@ export function createSecretContextComposable (options = {}) {
     return normalizeSecretManifest(object)
   })
 
-  const secretManifest = ref({})
+  const manifest = ref({})
```

Comment:
similar as with the renaming in `useCredentialsBindingContext` and `useSecretBindingContext`, rename `initialSecretManifest` -> `initialManifest`